ocaml/src/fragment_cache.ml

```diff
@@ -32,8 +32,11 @@ class virtual cache = object(self)
     method add' bid oid blob =
       self # add bid oid blob >>= fun _ -> Lwt.return_unit
 
-    method virtual lookup : int64 -> string -> (Lwt_bytes.t * (Manifest.t * int64 * string) list) option Lwt.t
-    method virtual lookup2 : int64 -> string
+    method virtual lookup : ?timeout:float
+                            -> int64 -> string
+                            -> (Lwt_bytes.t * (Manifest.t * int64 * string) list) option Lwt.t
+    method virtual lookup2 : ?timeout:float
```

Comment:
Did you intend on using this extra argument? (Currently it's not used anywhere.)